ert3/evaluator/_evaluator.py

```diff
@@ -31,78 +104,61 @@ def _create_evaluator_tmp_dir(workspace_root: Path, evaluation_name: str) -> Pat
+TupleDictStrAny = Tuple[Dict[str, Any], ...]
 
-    futures = []
-    for input_ in step_config.input:
-        for iens, record in enumerate(inputs.ensemble_records[input_.record].records):
-            if storage_config.get("type") == "shared_disk":
-                transmitter = ert3.data.SharedDiskRecordTransmitter(
-                    name=input_.record,
-                    storage_path=pathlib.Path(storage_config["storage_path"]),
-                )
-            else:
-                raise ValueError(
-                    f"Unsupported transmitter type: {storage_config.get('type')}"
-                )
-            futures.append(transmitter.transmit_data(record.data))
-            transmitters[iens][input_.record] = transmitter
-    asyncio.get_event_loop().run_until_complete(asyncio.gather(*futures))
-    if isinstance(step_config, ert3.config.Unix):
-        for command in step_config.transportable_commands:
-            if storage_config.get("type") == "shared_disk":
-                transmitter = ert3.data.SharedDiskRecordTransmitter(
-                    name=command.name,
-                    storage_path=pathlib.Path(storage_config["storage_path"]),
-                )
-            else:
-                raise ValueError(
-                    f"Unsupported transmitter type: {storage_config.get('type')}"
-                )
-            with open(command.location, "rb") as f:
-                asyncio.get_event_loop().run_until_complete(
-                    transmitter.transmit_data([f.read()])
-                )
-            for iens in range(0, ensemble_size):
-                transmitters[iens][command.name] = transmitter
-    return dict(transmitters)
 
+def _function_jobs(step: Function) -> TupleDictStrAny:
+    return (
+        {
+            "name": step.function.__name__,
+            "executable": cloudpickle.dumps(step.function),
+        },
+    )
 
-def _prepare_output(
-    ee_config: Dict[str, Any],
-    step_config: Step,
-    evaluation_tmp_dir: Path,
-    ensemble_size: int,
-) -> Dict[int, Dict[str, RecordTransmitter]]:
-    tmp_input_folder = evaluation_tmp_dir / "output_files"
-    os.makedirs(tmp_input_folder)
-    storage_config = ee_config["storage"]
-    transmitters: Dict[int, Dict[str, ert3.data.RecordTransmitter]] = defaultdict(dict)
-
-    for output in step_config.output:
-        for iens in range(0, ensemble_size):
-            if storage_config.get("type") == "shared_disk":
-                transmitters[iens][
-                    output.record
-                ] = ert3.data.SharedDiskRecordTransmitter(
-                    name=output.record,
-                    storage_path=pathlib.Path(storage_config["storage_path"]),
-                )
-            else:
-                raise ValueError(
-                    f"Unsupported transmitter type: {storage_config.get('type')}"
-                )
-    return dict(transmitters)
+
+def _unix_jobs(step: Unix) -> TupleDictStrAny:
```

Comment:
Add a `factory` to that name, it's a bit generic now, not really indicating what it does and why it exists.

Reply:
Would this also count for _function_job, even tho it's always returning a list of one dict object.